Astro CLI 1.2 docs (#855)
astronomer/docs: software_versioned_docs/version-0.25/cli-quickstart.md

> If the installation was successful, you should see the version of the CLI that you installed in the output:

Reviewer: ```suggestion
When the installation is successful, CLI version information is provided in the output message:
```

Author: This is legacy text from 0.25, we should avoid editing at this time
